fundamentals/fundamentals-bio/src/main/scala/izumi/functional/bio/BIOCatsConversions.scala

```diff
@@ -142,9 +143,36 @@ object BIOCatsConversions {
     }
   }
 
-  class BIOCatsSync[F[+_, +_]](override val F: BIO[F]) extends BIOCatsBracket[F](F) with cats.effect.Sync[F[Throwable, ?]] {
+  trait BIOCatsParallel[F[+_, +_]] {
+    val P: cats.Parallel[F[Throwable, ?]]
+    def F: BIOParallel[F]
+  }
+
+  class BIOCatsSync[F[+_, +_]](override val F: BIO[F]) extends BIOCatsBracket[F](F) with BIOCatsParallel[F] with cats.effect.Sync[F[Throwable, ?]] {
```

Comment:
`BIO` does not inherit `BIOParallel`. should it? if not, should `BIOCatsSync` take `BIOParallel` as a param?

Reply:
@Adriani277 
No, you should inherit a new class *from* BIOCatsSync:
```scala
class BIOCatsParallel[F0[+_, +_]](override val F: BIOParallel[F0]) extends BIOCatsSync[F0](F) with cats.Parallel[F0[Throwable, ?]] {
  override type F[A] = M[A]
  ...
}
```

And create a new implicit priority level after `BIOToSync` & before `BIOAsyncToAsync`:

```scala
trait BIOCatsConversions8 extends BIOCatsConversions9 {
  @inline implicit final def BIOParallelToParallelF[+_, +_]](implicit F: BIOParallel[F]): cats.effect.Sync[F[Throwable, ?]] with cats.Parallel[F[Throwable, ?]] with S11 = new BIOCatsAsync[F](F)
}
```

(To get `S11` you'll need to add more levels to `SpecificityHelper.scala` and add `with S11` to `BIOCatsFunctor`)